packages/node_modules/@node-red/runtime/locales/en-US/runtime.json

```
@@ -104,7 +104,9 @@
             "error":"Error loading credentials: __message__",
             "error-saving":"Error saving credentials: __message__",
             "not-registered": "Credential type '__type__' is not registered",
-            "system-key-warning": "\n\n---------------------------------------------------------------------\nYour flow credentials file is encrypted using a system-generated key.\n\nIf the system-generated key is lost for any reason, your credentials\nfile will not be recoverable, you will have to delete it and re-enter\nyour credentials.\n\nYou should set your own key using the 'credentialSecret' option in\nyour settings file. Node-RED will then re-encrypt your credentials\nfile using your chosen key the next time you deploy a change.\n---------------------------------------------------------------------\n"
+            "system-key-warning": "\n\n---------------------------------------------------------------------\nYour flow credentials file is encrypted using a system-generated key.\n\nIf the system-generated key is lost for any reason, your credentials\nfile will not be recoverable, you will have to delete it and re-enter\nyour credentials.\n\nYou should set your own key using the 'credentialSecret' option in\nyour settings file. Node-RED will then re-encrypt your credentials\nfile using your chosen key the next time you deploy a change.\n---------------------------------------------------------------------\n",
+            "unencrypted" : "Using unecrypted credentials",
+            "encryptednotfound" : "Encrypted credentials not found"
```

Either kebab-case or camelCase - never alllowercase
```suggestion
            "encryptedNotFound" : "Encrypted credentials not found"
```